test/unit/options.test.js

```diff
@@ -0,0 +1,11 @@
+const ssgulnur = require('../../src/options');
+
+describe('test the main function with bad values', () => {
+  test('return false if bad argument is passed to a function', () => {
+    expect(ssgulnur('')).toBeFalsy();
+  });
+
+  test('check if calling a function with bad value will match snapshot', () => {
+    expect(ssgulnur('Bad value')).toMatchSnapshot();
```

Comment:
Here, I think it is better to use toBe or toThrow rather than a snapshot as it only returns undefined